db/migrate/20230221081006_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+
+      t.timestamps
```

Comment:
- [ ] Please 🙏🏽 add `t.integer :post_counter` column to register the number of posts by user.